<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Do you want to make a seat reservation?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="YesButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NoButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
